src/components/UserCollective.js

```diff
@@ -461,7 +468,20 @@ class UserCollective extends React.Component {
                 </section>
               )}
 
-              {Object.keys(memberOfByRole).map(role => role !== 'HOST' && this.renderRole(role))}
+              {Object.keys(memberOfByRole).map(role => {
```

Comment:
Sorry, I don't understand the logic here.

Can you refactor and/or add comments so that we can understand better the idea behind the change?